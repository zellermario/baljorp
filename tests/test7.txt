create SJ m 0

create Eszkimo e m
create Etel field m

digobject e
pass e