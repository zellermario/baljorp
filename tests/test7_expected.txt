Fields:
SJ m (1) - object: ?
	neighbours: []

Buildings:

Actors:
> Eszkimo e (heat: 5) on m
  Inventory: []